### PR TITLE
Fix TypeError: Cannot read property 'text' of undefined error.

### PR DESCRIPTION
**Pull request by heavenshell**
## TL;DR

Fix `TypeError: Cannot read property 'text' of undefined` error.

## Detail

When importing `relative path` like following, `typescript-simple` raised exception.

- <https://github.com/9renpoto/ts/blob/fix/filename/test/index.test.ts#L1>

    ```
    TypeError: Cannot read property 'text' of undefined
        at TypeScriptSimple.toJavaScript (/tmp/src/ts/node_modules/typescript-simple/index.js:162:28)
        at TypeScriptSimple.compile (/tmp/src/ts/node_modules/typescript-simple/index.js:71:25)
        at Object.loadTypeScript (/tmp/src/ts/node_modules/espower-typescript/index.js:19:22)
    ```

### Actual behavior

`service.getEmitOutput(fileName)` returns `relative path` and `absorute path`.

eg.
\#: see also https://github.com/power-assert-js/espower-typescript/issues/16#issue-149396537
    
/tmp/src/sample/test.ts

```typescript
import {test} from './dep'

test()
```
    
/tmp/src/sample/dep.ts

```typescript
export function test() {
    console.log('from ts')
}
```

- `service.getEmitOutput('test.ts')` returns `relative path`
    - `test.js`
- `service.getEmitOutput('dep.ts')` returns `absolute path`
    - `/tmp/src/sample/deps.js`
    
So, [this line](https://github.com/teppeis/typescript-simple/blob/master/index.ts#L169) compared relative path(`outputFileName`) with absolute path(`file.name`) and returns `undefined`.

## Solution

Convert `relative path` to `absolute path`.

## PoC

Please see our sample project.

1. git clone https://github.com/9renpoto/ts.git
1. git checkout fix/filename
1. npm install
1. npm test

\# note

`package.json`'s `typescript-simple` pointed to our PR branch.

If you delete this line, exception would raised.

## Misc

This PR would also solve following issue.

- <https://github.com/power-assert-js/espower-typescript/issues/16>

Please check and include this PR. 

Thank you.

### CODE REVIEW

**Comment by teppeis** [comment]
@heavenshell Thanks!
Your patch didn't fix this issue in Windows, but I released it with "known issues" in v5.0.3.

**Comment by heavenshell** [comment]
@teppeis Thank you for solving this issue.
